components/FullwidthSlider/src/stories/icons.stories.mdx

```diff
@@ -0,0 +1,87 @@
+export const DownloadIcon = () => {
+  return (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Hebben de icons geen html class nodig om de kleur eventueel te kunnen aanpassen in css?
Zie die fill='currentColor', die kun je dan targeten.